code/segmentedtickerplant/datastripe.q

```diff
@@ -47,12 +51,13 @@ subsegment:{[tbl;segid];
           :();
      ];
      filter:segmentfilter[tbl;segid];
-     if[filter~"";
+     $[tbl in .stpps.ignoredtables;[.stpps.suball[tbl]];
```

Comment:
Why is there square brackets around the true part?
Also would look better with extra line as:
$[conditional;
    true;
    false]